src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -109,7 +109,8 @@ export class AssetsTable extends React.Component {
   }
 
   getImageThumbnail(thumbnail) {
-    return thumbnail ? (<img src={`${this.props.courseDetails.base_url}${thumbnail}`} alt="Description not available" />) : 'Preview not available';
+    const baseUrl = this.props.courseDetails.base_url ? this.props.courseDetails.base_url : '';
+    return thumbnail ? (<img src={`${baseUrl}${thumbnail}`} alt="Description not available" />) : 'Preview not available';
   }
```

Comment:
Thumbnails show up in local studio now! 
<img width="1513" alt="screen shot 2017-11-07 at 10 25 50 am" src="https://user-images.githubusercontent.com/9357438/32501506-308afee8-c3a6-11e7-9119-12eb4995a12d.png">